Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
         stage ('Build') {
             steps {
                 sh 'bash prepare_pack.sh'
-                sh 'zip GeyserOptionalPack.mcpack -r . -x ".*" Jenkinsfile required_files.txt prepare_pack.sh'
+                sh 'zip GeyserOptionalPack.mcpack -j -r build/'
```

Comment:
The `-j` option here removes the directory listings and puts everything into the zip without structure and does not work.